src/Interpreters/Streaming/Aggregator.h

```diff
@@ -749,6 +749,9 @@ class Aggregator final
     /// 3) For deleted group:
     ///     @p retracted_result: add an elem <group_key, last_state> if not exists
     ///     @p result:           delete the <group_key> group
+    /// @return {should_abort, need_finalization} pair
```

Comment:
nit : Let's use `\param` style for parameter comments moving forward since it is conventional. For example

```
/// \columns ...
/// \row_begin ...
/// \return {should_abort, need_finalization} bool pair
```

Reply:
ok, and we can have a global sharing about these styles

Reply:
In fact, I have seen many different styles

Reply:
The style above is CLion / Doxygen etc can render